particles/Renderer.cpp

```diff
@@ -169,6 +173,10 @@ void Renderer::init(score::gfx::RenderList& renderer)
     {
         Speed d[];
     } sbuf;
+    layout(std140, binding = 2) uniform SMod
```

Comment:
perfect :) but maybe call it "Controls" instead of something like that. This way when we add more controls they can just go into the struct.

Likewise in the code above: 

```
    particleSpeedModifier = renderer.state.rhi->newBuffer(
                QRhiBuffer::Immutable, QRhiBuffer::UniformBuffer, sizeof(float));
```

what I'd recommend is to instead of having

```
float speedModifier;
```

to have 
```
struct Controls {
float speedModifier;
};
```
and do 

    particleSpeedModifier = renderer.state.rhi->newBuffer(
                QRhiBuffer::Immutable, QRhiBuffer::UniformBuffer, sizeof(Controls));
                
                
this way, when adding new controls, you can just add variables in the struct in the CPP and in the shader, and that should work without having to update the sizes everywhere